1.2: Update the cli command reference documentation directory

Caleb-Hill opened the pull request:
Updates to cli command reference documentation directory.

Caleb-Hill committed:
Fix sawtooth.md

Fill in the cli reference outputs and fix markdown formatting issues.

Signed-off-by: Caleb Hill <hill@bitwise.io>

Caleb-Hill committed:
Fix sawtooth-validator.md

Fill in cli output help messages and markdown formatting fixes.

Signed-off-by: Caleb Hill <hill@bitwise.io>

Caleb-Hill committed:
Fix sawtooth-rest-api.md

Fill in cli output help messages and markdown formatting fixes.

Signed-off-by: Caleb Hill <hill@bitwise.io>

Caleb-Hill committed:
Fix sawadm.md

Fill in cli output help messages and markdown formatting fixes.

Signed-off-by: Caleb Hill <hill@bitwise.io>

Caleb-Hill committed:
Resolve commited merge conflict in sawadm.md

A merge conflict git artifact got commited sometime in the past, this
commit removes it and sets the text to be the same as it is currently on
the website.

Signed-off-by: Caleb Hill <hill@bitwise.io>

Caleb-Hill committed:
Fix sawnet.md

Fill in cli output help messages and markdown formatting fixes.

Signed-off-by: Caleb Hill <hill@bitwise.io>

Caleb-Hill committed:
Fix sawset.md

Fill in cli output help messages and markdown formatting fixes.

Signed-off-by: Caleb Hill <hill@bitwise.io>

vaporos commented on docs/core/1.2/cli/index.md:
```diff
@@ -0,0 +1,98 @@
+# CLI Command Reference
```
Duplicate header

Caleb-Hill replied:
Fixed

vaporos commented on docs/core/1.2/cli/sawadm.md:
```diff
@@ -35,34 +51,23 @@ initialization of a validator. A network requires an initial block
-`--ignore-required-settings` argument is used (run
-`sawadm genesis --help` for more info). \>\>\>\>\>\>\>
-core/1-2:docs/core/1.2/cli/sawadm.rst
+The [input_file]{.title-ref} argument specifies one or more files containing
```
.title-ref is a sphinx-ism I think

Caleb-Hill replied:
Fixed in all files I had already worked in

vaporos reviewed (changes requested):
Needs to be moved an hooked up the the sidebar.

Caleb-Hill committed:
Move cli directory from core to 1.2

Partially a rename partially a modification because some of the files existed
before.

Signed-off-by: Caleb Hill <hill@bitwise.io>

Caleb-Hill committed:
Add links for cli docs to sidebar

Fixes links so they point at the relevant cli documenation files.

Signed-off-by: Caleb Hill <hill@bitwise.io>

rbuysse commented on docs/core/1.2/cli/sawtooth-validator.md:
````diff
@@ -35,9 +35,9 @@ validator.
-::: literalinclude
-output/sawtooth-validator_usage.out
-:::
+``` console
````
Looks like this got missed

```
sawtooth-validator 1.2.6
Configures and starts a Sawtooth validator.

USAGE:
    sawtooth-validator [FLAGS] [OPTIONS]

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information
    -v, --verbose    enable more verbose output to stderr

OPTIONS:
    -B, --bind <bind>...
            set the URL for the network or validator component service endpoints with the format network:<endpoint>,
            component:<endpoint>, or consensus:<endpoint>. Use multiple --bind options to specify all endpoints.
        --config-dir <config_dir>                                  specify the configuration directory
    -E, --endpoint <endpoint>                                      specifies the advertised network endpoint URL
        --fork-cache-keep-time <fork_cache_keep_time>              set the time in seconds to keep uncommitted forks.
        --maximum-peer-connectivity <maximum_peer_connectivity>    set the maximum number of peers to accept
        --minimum-peer-connectivity <minimum_peer_connectivity>
            set the minimum number of peers required before stopping peer search

        --network-auth <network_auth>
            identify type of authorization required to join validator network. [possible values: trust, challenge]

        --opentsdb-db <opentsdb-db>                                specify name of database used for storing metrics
        --opentsdb-url <opentsdb-url>
            specify host and port for Open TSDB database used for metrics

    -P, --peering <peering>
            determine peering type for the validator: 'static' (must use --peers to list peers) or 'dynamic' (processes
            any static peers first, then starts topology buildout). [possible values: static, dynamic]
    -p, --peers <peers>...
            list static peers to attempt to connect to in the format tcp://<hostname>:<port>. Specify multiple peers in
            a comma-separated list. Repeating the --peers option is also accepted.
        --scheduler <scheduler>
            set scheduler type: serial or parallel [possible values: serial, parallel]

    -s, --seeds <seeds>...
            provide URI(s) for the initial connection to the validator network, in the format tcp://<hostname>:<port>.
            Specify multiple URIs in a comma-separated list. Repeating the --seeds option is also accepted.
        --state-pruning-block-depth <state_pruning_block_depth>
            set the block depth below which state roots are pruned from the global state database.
```
